app/inputsearchmodel.cpp

```diff
@@ -0,0 +1,20 @@
+#include "inputsearchmodel.h"
+
+InputSearchModel::InputSearchModel( QObject *parent )
```

Comment:
`=default` instead of brackets

Reply:
Default constructor cannot have arguments. The InputSearchModel is inherited from QAbstractListModel where a constructor has param `QObject *parent`